src/pudl/transform/ferc1.py

```diff
@@ -414,13 +409,13 @@ def assign_utility_id_ferc1(
 
 
 class FuelFerc1TableTransformer(Ferc1AbstractTableTransformer):
-    """A table transformer specific to the ``fuel_ferc1`` table.
+    """A table transformer specific to the :ref:`fuel_ferc1` table.
 
-    The ``fuel_ferc1`` table reports data about fuel consumed by large thermal power
-    plants that report in the ``plants_steam_ferc1`` table.  Each record in the steam
-    table is typically associated with several records in the fuel table, with each fuel
-    record reporting data for a particular type of fuel consumed by that plant over the
-    course of a year. The fuel table presents several challenges.
+    The :ref:`fuel_ferc1` table reports data about fuel consumed by large thermal power
+    plants in the :ref:`plants_steam_ferc1` table. Each record in the steam table is
+    typically associated with several records in the fuel table, with each fuel record
+    reporting data for a particular type of fuel consumed by that plant over the course
+    of a year. The fuel table presents several challenges.
```

Comment:
I think *a lot* of the content below this (not showing up in git) is duplicated information at this point and the meat of the information can/should/does live near the transformers themselves.

Reply:
if it were up to me, I would communicate here in what ways the fuel table is unique and needs unique staging of transforms.

For instance, cleaning the freeform strings of the PK columns need to happen earlier than normal so `process_dbf` is special.

Reply:
Hmm, we've had several conversations about where to accumulate / document a more descriptive narrative of wtf is going on with each table, more for reference and feedback fo FERC than as "traditional" docstrings that are talking more about the code or function at hand, and we landed on using the TableTransformer class docstrings, since these are the classes that organize together all of the table-specific shenanigans.

Maybe we should look at the rendered docs and see if breaking out this content into the individual transform function gives the same kind of overview, or if it feels jumbled.

Reply:
hm okay... i guess this is fine for now but I'd love to rethink how we are documenting the table transformations in light of this new infrastructure. When we can daylight the specific transform parameters or sequencing of the transformers it seems kinds silly to be very verbose here.

but i could be convinced otherwise and i think this is fine for now.